selftests/functional/test_teststmpdir.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import tempfile
+import shutil
+
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from avocado.core import exit_codes
+from avocado.utils import process
+from avocado.utils import script
+
+
+basedir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
+basedir = os.path.abspath(basedir)
+
+
+SIMPLE_SCRIPT = """#!/bin/bash
+touch ${AVOCADO_TESTS_COMMON_TMPDIR}/file1.txt
```

Comment:
Well this passes even when you run the test multiple times. How about `mktemp $AVOCADO_TESTS_COMMON_TMPDIR/simple_XXXX.txt`